DRAFT.md

```diff
@@ -1089,31 +1101,22 @@ constexpr indirect& operator=(indirect&& other) noexcept(
     allocator_traits<Allocator>::is_always_equal::value);
 ```
 
-5. _Mandates_: If `allocator_traits<allocator_type>::is_always_equal::value`
+5. _Mandates_: If `allocator_traits<Allocator>::is_always_equal::value`
     is `false`, then `T` is a complete type.
 
-6. _Effects_: If `addressof(other) == this` is `true`, there are no effects.
+6. _Effects_: If `addressof(other) == this` is `true`, there are no effects. Otherwise:
 
-  If`allocator_traits<allocator_type>::propagate_on_container_move_assignment` is
-  `true` then the allocator needs updating.
+    6.1. The allocator needs updating if `allocator_traits<Allocator>::propagate_on_container_move_assignment` is `true`.
 
-  If `other` is valueless, `*this` becomes valueless and the owned object in
-  `*this`, if any, is destroyed using `allocator_traits<allocator_type>::destroy`
-  and then the storage is deallocated.
+    6.2. If `other` is valueless, `*this` becomes valueless and the owned object in `*this`, if any, is destroyed using `allocator_traits<Allocator>::destroy` and then the storage is deallocated.
 
-  Otherwise, if `alloc == other.alloc` is `true`, swaps the owned objects in `*this` and
-  `other`; the owned object in `other`, if any, is then destroyed using
-  `allocator_traits<allocator_type>::destroy` and then the storage is deallocated.
+    6.3. Otherwise, if `alloc == other.alloc` is `true`, swaps the owned objects in `*this` and `other`; the owned object in `other`, if any, is then destroyed using `allocator_traits<Allocator>::destroy` and then the storage is deallocated.
 
-  Otherwise constructs a new owned object with the owned object
-  `other` as the argument as an rvalue, using either the allocator in `*this` or
-  the allocator in `other` if the allocator needs updating. The previous owned
-  object in `*this`, if any, is destroyed using
-  `allocator_traits<allocator_type>::destroy` and then the storage is
-  deallocated.
+    6.4. Otherwise constructs a new owned object with the owned object `other` as the argument as an rvalue, using either the allocator in `*this` or the allocator in `other` if the allocator needs updating.
 
-  If the allocator needs updating, the allocator in `*this` is
-  replaced with a copy of the allocator in `other`.
+    6.5. The previously owned object in `*this`, if any, is destroyed using `allocator_traits<Allocator>::destroy` and then the storage is deallocated.
+
+    6.6. If the allocator needs updating, the allocator in `*this` is replaced with a copy of the allocator in `other`.
```

Comment:
Similarly to above, should "replaced with a copy of the allocator in `other`" be reworded here to make it clearer whether it's copy-assignment?

Reply:
Left intentionally vague for some reason!